chain-impl-mockchain/src/ledger/ledger.rs

```diff
@@ -381,6 +389,12 @@ impl Ledger {
         if total_reward > Value::zero() {
             let total_blocks = leaders_log.total();
             let reward_unit = total_reward.split_in(total_blocks);
+            let mut pool_count = 0;
+
+            // Is there a neater way to do this?
+            for (pool_id, pool_blocks) in leaders_log.iter() {
```

Comment:
`leaders_log.iter().len()`

Reply:
Thanks!

Reply:
len() doesn't seem to exist.  I have used count() instead, which is a bit cleaner than a loop.

Reply:
my apologies for the mis-suggestion, I meant `count()` indeed ! glad you manage to find it.